group-coordinator/src/test/java/org/apache/kafka/coordinator/group/OffsetMetadataManagerTest.java

```diff
@@ -177,6 +183,29 @@ public CoordinatorResult<OffsetCommitResponseData, Record> commitOffset(
             return result;
         }
 
+        public CoordinatorResult<OffsetDeleteResponseData, Record> deleteOffsets(
+            OffsetDeleteRequestData request
+        ) {
+            snapshotRegistry.getOrCreateSnapshot(lastWrittenOffset); // TODO: lastCommitedOffset or lastWrittenOffset?
```

Comment:
In the GroupMetadataManagerTestContext, we actually moved this to the replay method. See [here](https://github.com/apache/kafka/blob/ad7956170bcaf093ea8b2f725126d42cf7fb522b/group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java#L1144). It may be better to do the same here. What do you think?